src/collective/fingerpointing/upgrades/configure.zcml

```diff
@@ -0,0 +1,27 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:gs="http://namespaces.zope.org/genericsetup"
+    i18n_domain="collective.fingerpointing">
+
+    <gs:registerProfile
+        name="1_to_2"
+        title="Upgrade collective.fingerpointing GS profile to 2"
+        directory="profiles/1_to_2"
+        description=""
+        provides="Products.GenericSetup.interfaces.EXTENSION"
+        for="Products.CMFPlone.interfaces.IMigratingPloneSiteRoot"
+        />
+
+    <gs:upgradeSteps
+        source="1"
+        destination="2"
+        profile="collective.fingerpointing:default">
+
+        <gs:upgradeDepends
+            title="Add the audit_logfile registry setting."
+            import_profile="collective.fingerpointing.upgrades:1_to_2"
```

Comment:
Hmm not sure, this is how i learned to to do Upgrade profiles. For example p.a.querystring does it also like that: https://github.com/plone/plone.app.querystring/blob/master/plone/app/querystring/upgrades.zcml